4-WebApp-your-API/4-3-AnyOrg/ToDoListClient/appsettings.json

```diff
@@ -17,15 +17,14 @@
   },
   "TodoList": {
     /*
-      TodoListScope is the scope of the Web API you want to call. This can be: "api://b10faa7f-02e8-451c-a405-2a7e9e7072f0/.default",
+      TodoListScope is the scope of the Web API you want to call. This can be: "api://f26c94a8-713b-46c4-869a-461fbb209f35/.default",
       - a scope for a V2 application (for instance api://b3682cc7-8b30-4bd2-aaba-080c6bf0fd31/access_as_user)
       - a scope corresponding to a V1 application (for instance <GUID>/user_impersonation, where  <GUID> is the
         clientId of a V1 application, created in the https://portal.azure.com portal.
     */
     "TodoListAppId": "[Enter_client_ID_Of_TodoListService-v2_from_Azure_Portal,_e.g._2ec40e65-ba09-4853-bcde-bcb60029e596",
     "TodoListScope": "api://[Enter_client_ID_Of_TodoListService-v2_from_Azure_Portal,_e.g._2ec40e65-ba09-4853-bcde-bcb60029e596]/.default",
-    //api://83ade765-7b4f-4208-b69f-c28a2b4bebec/.default",
-    "TodoListBaseAddress": "https://localhost:44351",
+    "TodoListBaseAddress": "https://localhost:44351/",
```

Comment:
Can we rename these properties? It's not clear if you are talking about TodoListClient or TodoListService here